apps/labs/posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Quansight Labs awarded three CZI EOSS Cycle 5 Grants"
+author: jaime-rodriguez-guerra
+published: November 10, 2022
+description: "We are delighted to share details about new grants to support the sustainability of SciPy, conda-forge, and CuPy"
+category: [Funding, PyData Ecosystem, Packaging, Community]
+featuredImage:
+  src: /posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants/blog_feature_czi5.svg
+  alt: ‘Logos of CZI, conda-forge, scipy, and CuPy'
+hero:
+  imageSrc: /posts/quansight-labs-awarded-three-czi-eoss-cycle5-grants/blog_hero_czi5.svg
```

Comment:
Somehow SciPy's logo is cut which is not good. And Conda-Forge is blurry (does not seem to be happening for the featuredImage on the blog posts list)

Reply:
I don't have a higher res version of the conda-forge logo. Maybe @jaimergp can help with that? This is what I could get from their repository and website.
The SciPy logo is only cut in the preview due to a known bug #555. It should be ok in the live version.